newspeek/src/test/java/use_case/populate_list_with_articles/PopulateListOutputDataTest.java

```diff
@@ -0,0 +1,19 @@
+package use_case.populate_list_with_articles;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+class PopulateListOutputDataTest {
+    @Test
+    void successTest() {
```

Comment:
Simple enough. Good idea to use a generic HashMap instead of trying to mock article IDs and titles.